core/src/commonMain/kotlin/com/ustadmobile/core/view/PersonAccountEditView.kt

```diff
@@ -0,0 +1,33 @@
+package com.ustadmobile.core.view
+
+import com.ustadmobile.lib.db.entities.Person
+
+interface PersonAccountEditView: UstadEditView<Person> {
+
+    var firstPassword: String?
+
+    var secondPassword: String?
+
+    var secondPasswordFieldRequiredErrorVisible: Boolean
+
+    var firstPasswordFieldRequiredErrorVisible: Boolean
```

Comment:
Couldn't we have passwordError: String? and confirmPasswordError: String?, then a null value means no error.